.golangci.yml

```diff
@@ -14,3 +14,7 @@ linters:
     - godox
     - gomnd
     - gocognit
+
+issues:
+  exclude:
+    - Using the variable on range scope .* in function literal
```

Comment:
let's please fix the issue instead of exclude it...

Reply:
I have never seen that error before and navigating through the Github, I found most of people "solving" their problems with that.

Sorry, my bad

Reply:
no problem :)